cloudhub-2/modules/ROOT/pages/ch2-config-endpoints-paths.adoc

```diff
@@ -140,6 +140,9 @@ http://helloch2:8081/hello
 ```
 If required, replace `http` with `https`.
 
+[NOTE]
+You can't send traffic between applications in different namespaces using `svc.cluster.local` endpoints.
```

Comment:
Depending on the meaning, "using" should be changed to either "by using" or "that use".